### DISCUSSION TITLE
Office Hours

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#34] [Announcements]
<p align="center">
<img src="https://github.com/json-schema-org/community/assets/40007659/d01e323e-f32c-48f8-8230-9e58a765347c" width="500">
</p>

# We will be now meeting weekly! 🎉

"Office Hours" provide time for community members to ask any questions, discussions, comments, or just to chat about JSON Schema, via Google Meet.

Everyone is welcome. Registration not required. Calls are recorded but recordings are not public.

**Time: Every Tuesday, 18:00 UTC**
Meeting link: [meet.google.com/rxu-aazi-ddj](https://meet.google.com/rxu-aazi-ddj)

You can find these events scheduled on our **[JSON Schema Calendar](https://calendar.google.com/calendar/embed?src=info%40json-schema.org&ctz=Africa%2FLagos)**.

To add the events to your Google Calendar, use the link found in the bottom right of the above calendar.
To add the events to another calendar, use the provided [ics/ical file](https://calendar.google.com/calendar/ical/info%40json-schema.org/public/basic.ics).

All calls are subject to our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).

**Comment by ssbarnea** [comment]
Please add a calendar (ics) link here so can easily add it to our calendars.

**Reply by Relequestual** [reply]
Unfortunatley the one provided by Zoom isn't setup correctly for timezone changes.
I'm going to adjust the format to only be once per month, and allow for additional ad-hoc sessions as requested.
Stay tuned =]

**Comment by benjagm** [comment]
Hi everyone, we needed to update the Calendar urls because of upgrading our gmail account for our community meetings.  New URLs are:
- Calendar: https://calendar.google.com/calendar/embed?src=json.schema.community%40gmail.com&ctz=Europe/London
- iCal: https://calendar.google.com/calendar/ical/json.schema.community%40gmail.com/public/basic.ics

Apologies for the inconvenience.

**Reply by JasonPolis-ISDA** [reply]
Yes, please update the calendars, front page and invites and  to be consistent.

Where the meets in this issue point to zfu-mafp-qsp and aun-brma-sfe
the ics invites list iku-areu-ide and dcx-ijyy-ahd

The ics invites show November at 23:00Z and December at 15:00Z.

Upcoming events on home page says
JSON Schema Office Hours (APAC/Americas) November 5th 2024, 3:00 pm (UTC)

Google Calendar shows 11pm UTC today, but no one else has joined any of the 4 meets,
and also says _Every First Tuesday 15:00 BST, and by appointment!_

The [appointment url ](https://calendly.com/ben-hutton-postman/office-hours-request) is not valid

**Reply by benjagm** [reply]
> Yes, please update the calendars, front page and invites and to be consistent.

Thanks @JasonPolis-ISDA . We have an issue opened to fix the front page timezones: https://github.com/json-schema-org/website/issues/1036

**Comment by throwbackman98-collab** [comment]
response.status
